examples/custom_model_runner_chat_gpt.ipynb

```diff
@@ -2,151 +2,173 @@
  "cells": [
```

Comment:
Question (non-blocking): Do we no longer have an example notebook for toxicity evaluation?

Reply:
I deleted the "Toxicity evaluation using JumpStart model" example notebook since it's basically a copy-paste of the "Stereotyping evaluation using JumpStart model" notebook.